frontend/src/app/case-studies/case-study-view/footnotes/footnotes.directive.ts

```diff
@@ -0,0 +1,39 @@
+import { Directive, ElementRef, HostListener } from "@angular/core";
+
+/**
+ * Directive to fix hash links for footnotes
+ */
+@Directive({
+    selector: '[lcFootnotes]',
+    standalone: false,
+})
+export class FootnotesDirective {
+    constructor(
+        private el: ElementRef,
```

Comment:
```suggestion
        private el: ElementRef<HTMLElement>,
```

You will then see that `findElement` has a type error: `querySelector` does not return `undefined` if it can't find anything, but `null`.